### PR TITLE
Fix tags for docker images

### PR DESCRIPTION
**Pull request by mathieupotier**
Be careful potential BC break for those who are stickied to latest tag.

### CODE REVIEW

**Comment by mathieupotier** [comment]
I noticed that the latest tag is not the expected one.

image

Potential BC Break, but I dealed myself with backups to restrore previous expected version ...

File: update.sh [hunk 1]
```diff
@@ -2,27 +2,20 @@
 
 set -e
 
-DOCKER_BUILD=${DOCKER_BUILD:-0}
+DOCKER_BUILD=${DOCKER_BUILD:-1}
```
Comment (tuxgasy) [inline update.sh:5]: Why set DOCKER_BUILD to 1 by default now? It's better to leave it at zero.

File: update.sh [hunk 2]
```diff
@@ -82,23 +75,26 @@ for dolibarrVersion in ${DOLIBARR_VERSIONS[@]}; do
...
   fi
 
-  tags="${tags}${dolibarrVersion}"
-  if [ "${dolibarrVersion}" != "develop" ]; then
-    tags="${tags} ${dolibarrMajor}"
+  if [ ${dolibarrVersion} = "develop" ]; then
+    tags="${tags} develop"
+  else
+    tags="${tags} ${dolibarrVersion} ${dolibarrMajor}"
+  fi
```
Comment (tuxgasy) [inline update.sh:94]: In the generated README, there are double spaces in the tag list. Just remove the space after ${tags}.

**Comment by mathieupotier** [comment]
Done, thanks for review

**Comment by mathieupotier** [comment]
Too fast !! thanks @tuxgasy ^_^

**Pull request merged**